tests/ibr/test_service.py

```diff
@@ -0,0 +1,307 @@
+#######################################################################
+# File name: test_service.py
+# This file is part of: aioxmpp
+#
+# LICENSE
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+########################################################################
+import unittest
+import unittest.mock
+import aioxmpp
+import asyncio
+
+import aioxmpp.ibr.service as ibr_service
+import aioxmpp.ibr.xso as ibr_xso
+
+from aioxmpp.testutils import (
+    make_connected_client,
+    run_coroutine,
+)
+
+
+TEST_PEER = aioxmpp.JID.fromstr("juliet@capulet.lit/balcony")
+
+
+class TestService(unittest.TestCase):
```

Comment:
Good job on following the conventions & patterns used in the other test modules here! :+1: